intro.qmd

```diff
@@ -1,6 +1,16 @@
 # Introduction
 
 Making your first contribution to Open Source can be both empowering and yet very intimidating.
+
+A study[^1] analyzing the social barriers preventing newcomers from making their first contributions found that the three most important barriers are: 
+
```

Comment:
I think this edit more accurately reflects where these ideas come from. The original statement sounded like these three points were their conclusion, but I think these points are actually your ideas (based on their work), not their conclusions :) 

```suggestion
Steinmacher et al.[^1] identified 13 social barriers preventing newcomers from making their first contributions to open source software projects, including barriers related to Reception, Communication, and Orientation. We consider that most of these barriers are, in one way or another, related to: 

```